### PR TITLE
Minor refactoring in namespace label tests

### PR DESCRIPTION
**Pull request by mkoppal-px**
**What this PR does / why we need it**:

- [x] Renamed `ManualBackupAndScheduleBackupUsingNamespaceLabel` to `ManualAndScheduledBackupUsingNamespaceAndResourceLabel`
- [x] Moved to `ManualAndScheduleBackupUsingNamespaceLabel` to `backup_namespaced_labelled_test.go`
- [x] Removed the single quotes from `ScheduleBackupWithAdditionAndRemovalOfNS`

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: tests/backup/backup_namespace_labelled_test.go [hunk 1]
```diff
@@ -1207,3 +1207,304 @@ var _ = Describe("{ManualAndScheduleBackupUsingNSLabelWithMaxCharLimit}", func()
 		CleanupCloudSettingsAndClusters(backupLocationMap, credName, cloudCredUID, ctx)
 	})
 })
+
+// ManualAndScheduleBackupUsingNamespaceLabel Perform Namespace labeled manual and schedule backup of single and multiple namespaces along with default and custom restore
```
Comment (kphalgun-px) [inline tests/backup/backup_namespace_labelled_test.go:1211]: How about // ManualAndScheduleBackupUsingNamespaceLabel performs?

Comment (mkoppal-px) [reply tests/backup/backup_namespace_labelled_test.go:1211]: It is written to convey what is is doing

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Review by vpinisetti-px — APPROVED** [review]
LGTM

**Review by ak-px — APPROVED** [review]
LGTM

**Pull request merged**